CHANGELOG.md

```diff
@@ -3,22 +3,23 @@
 ## unreleased
 
 ### Improvements
-+ Add workflow for examples
-+ New package `Web`, It can be used in Dapps to interact with ASAs and Apps
-    + Compatible with `algosigner`.
-    + Support `executeTransaction` function with web package
-+ Move Error lists, BuilderError, mkTransaction to web package
-+ Reuse mkTransaction, Errors in algob and runtime
++ Added github workflows/examples.yaml to execute [`/examples`](https://github.com/scale-it/algo-builder/tree/master/examples) on a private net, on push.
++ Moved [error](http://algobuilder.dev/api/runtime/modules/errors.html) lists, BuilderError, [mkTransaction](http://algobuilder.dev/api/runtime/modules.html#mktransaction) to `@algo-builder/web` package. Reuse mkTransaction, errors in algob and runtime from `@algo-builder/web`.
 + Updated `algob init` to initialize a typescript project as well by passing `--typescript` flag. Usage: `algob init <location> --typescript`.
-+ Support pooled transaction fees in algob and runtime. Added tests
++ Support pooled transaction fees in algob and runtime. Added tests.
 + Add `flatFee` in `TxParams`
-+ Added support for teal debugger (get dryrun response or start debugger in chrome) in `algob` scripts.
-+ User can initialize & use accounts by name in runtime, similar to algob ('john', 'bob' etc)
-+ Updated `algob sign-multisig`:
++ Added support for teal debugger (get dryrun response or start debugger using `tealdbg` in chrome) in `algob` scripts.
++ User can initialize & use accounts by name in `@algo-builder/runtime`, similar to algob ('john', 'bob' etc)
++ Updates to `algob sign-multisig`:
     * Creating a new multisigned transaction (requires multisig metadata: `v, thr, addrs`)
     * Support for signing in a group transaction (loaded from file).
```

Comment:
could you add a short example or a link to an example?

Reply:
ok, adding link to /guide where we explain usage with example.